python/cudf/cudf/core/multiindex.py

```diff
@@ -156,7 +156,7 @@ def __init__(
 
         source_data = {}
         for i, (column_name, col) in enumerate(codes._data.items()):
-            if -1 in col.values:
+            if -1 in col:
```

Comment:
Made this change for perf reasons:
```python
In [22]: s = cudf.Series(range(0, 10000))

In [23]: x = s.values

In [24]: -1 in x
Out[24]: False

In [26]: %timeit (-1 in s.values)
662 ms ± 9.53 ms per loop (mean ± std. dev. of 7 runs, 1 loop each)

In [27]: %timeit (-1 in s)
1.93 µs ± 24.8 ns per loop (mean ± std. dev. of 7 runs, 1,000,000 loops each)
```

Reply:
Which reminds me that https://github.com/rapidsai/cudf/issues/11648 is still open